encrypted-media-respec.html

```diff
@@ -134,15 +134,14 @@
       <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/192">Issue 192</a> - The specification should do more to encourage/ensure CDM-level interoperability.</p>
       <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/221">Issue 221</a> - This specification contains sections for describing <a href="#security">security</a> and <a href="#privacy">privacy</a> considerations. These sections are not final and review is welcome.</p>
 
-<!-- This will be populated when addressing https://github.com/w3c/encrypted-media/issues/206.
       <p>The following features are <strong>at risk</strong> and may be removed due to lack of implementation.
       </p>
       <ul>
-        <li><a def-id=""></a></li>
+        <li><a def-id="persistent-usage-record-session"></a></li>
+        <li><a def-id="media-key-session-destroyed-algorithm"></a></li>
```

Comment:
Similarly, `The <a def-id="media-key-session-destroyed-algorithm"></a> algorithm`